dockers/tpu-tests/common.libsonnet

```diff
@@ -0,0 +1,118 @@
+local base = import 'templates/base.libsonnet';
+local metrics = import 'templates/metrics.libsonnet';
+local volumes = import 'templates/volumes.libsonnet';
+
+{
+  local CloudAcceleratorTest = base.BaseTest {
+    local config = self,
+
+    publisherImage: 'gcr.io/xl-ml-test/publisher:stable',
+
+    tpuSettings+: {
+      requireTpuAvailableLabel: true,
+    },
+
+    metricConfig:
+      metrics.CompatMetrics(config.metricCollectionConfig, config.regressionTestConfig),
+
+    // Add experimental TPU health monitor to Job.
+    podTemplate+:: {
+      spec+: {
+        containerMap+: if config.accelerator.type == 'tpu' then
+          {
+            monitor: {
+              name: 'monitor',
+              image: 'gcr.io/xl-ml-test/health-monitor:stable',
+              imagePullPolicy: 'Always',
+              env: [
+                {
+                  name: 'POD_NAME',
+                  valueFrom: {
+                    fieldRef: {
+                      fieldPath: 'metadata.name',
+                    },
+                  },
+                },
+                {
+                  name: 'POD_NAMESPACE',
+                  valueFrom: {
+                    fieldRef: {
+                      fieldPath: 'metadata.namespace',
+                    },
+                  },
+                },
+              ],
+            },
+          }
+        else {},
+      } + if config.accelerator.type == 'gpu' then {
+        priorityClassName: 'gpu-%(version)s' % config.accelerator,
+      } else if config.accelerator.type == 'tpu' && std.endsWith(config.testName, '-1vm') then {
+        priorityClassName: if config.accelerator.replicas == 1 then
+          'tpu-device' else 'tpu-pod',
+      } else {},
+    },
+
+    cronJob+:: {
+      metadata+: {
+        namespace: 'automated',
+      },
+    },
+  },
+  local PyTorchBaseTest = CloudAcceleratorTest {
+    configMaps+: ['pytorch-nfs-ip'],
+    regressionTestConfig+: {
```

Comment:
you'll want to remove the regressionTestConfig since I think you're not using a bigquery backend and tracking metrics over time like we are